frontend/src/Components/ToastAdded/ToastAlreadyAdded.js

```diff
@@ -0,0 +1,10 @@
+// toastAdded.js
+import Swal from 'sweetalert2';
```

Comment:
this component is not reusable, how about if i need to have 100 toast, you will declare 100 component? 
instead of you only need to create 1 component and pass properties (title, text, icon, etc...) as prop

Reply:
Can you combine `ToastAlreadyAdded.js` and `ToastNewAdded.js` ?